Migrated and updated Extending packages articles (#14)
umbraco/UmbracoCMSDocs: Articles/Packages/Extending/uploading-to-our.md

### Uploading the package to Our

To get started head over to your [profile page](https://our.umbraco.com/member/profile/). It will tell you on this page if you have enough karma to upload your package, if you do then click `Add Package`.

![Your packages profile page](images/PackagesPage.png)

You may notice if you followed the [Creating a Package](../Creating-a-Package/) document that a lot of this information has already been entered. This isn't automatically pulled from the package file and will need to be entered again.

The form will save each page as you go and the package won't become live until you finish the process and mark it live on the final page. This means you can come back and finish your package submission at any time.
	
## Package Details

| Property | Value | Note |
| -------- | ----- | ---- |
| Title | My Package | The name of your package |
| Current version | x.x.x | The version of your package. This will display on the front end in the button (regardless of file name) |
| Package Category | E.g. Backoffice Extension | The category this package should be put into. |

> | Package Description | This package will extend the backoffice. | What your package does. Be clear and to the point about what your package does. |

Reviewer: ```suggestion
| Package Description | This package will extend the backoffice. | Be clear and to the point about what your package does. |
```

| License name | MIT License | As input in Umbraco |
| License URL | http://opensource.org/licenses/MIT | As input in Umbraco |

### Resources

| Property | Value | Note |
| -------- | ----- | ---- |

> | Package URL | https://umbraco.com | This url will be shown as the "packages url" when others install it. Will mostly be a Github repo or similar. |

Reviewer: ```suggestion
| Package URL | https://umbraco.com | This URL will be shown as the package's URL when others install it. Will likely be a Github repo, or similar. |
```

| Demonstration URL | https://umbraco.com | If you have a video, blog, audio description of how your package works in more depth, this is the place to advertise it. |
| Source code URL | https://umbraco.com | A link to the source code of the package. |
| NuGet package URL | | A link to the NuGet listing of your package if available. |
| Bug tracking URL | | If you want people to submit bugs in a specific place, pop the link in here. |

### Analytics and tracking

| Property | Value | Note |
| -------- | ----- | ---- |

> | Google Analytics code | UA-11111111-1 | This will tie in stats about your package into your Google Analytics account.

Reviewer: ```suggestion
| Google Analytics code | UA-11111111-1 | This will include statistics from your package into your Google Analytics account.
```

Author: This sounds a bit off to me, maybe "This will include statistics from your package in your Google Analytics account."? 🤔

Reviewer: Yes, sorry - I meant just "include".  Have updated the suggestion.

### Collaboration

| Property | Value | Note |
| -------- | ----- | ---- |

> | This package is open for collaboration | True/false | This currently sets an internal flag for the package and doesn't show any different on the front end |

Reviewer: ```suggestion
| This package is open for collaboration | True/false | This currently sets an internal flag for the package and doesn't lead to any difference in display of the package details to other users. |
```

### Package status

| Property | Value | Note |
| -------- | ----- | ---- |
| This package is retired | _Tick_ | Tick this box if you are no longer actively maintaining the package. |

> | Retired message | I no longer have time to work on this package. | A quick message to explain why you are not maintaining the package. This will be shown to your Our package listing |

Reviewer: ```suggestion
| Retired message | I no longer have time to work on this package. | A quick message to explain why you are not maintaining the package. This will be shown to your Our package listing. |
```
